Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -937,24 +937,25 @@ AXTextMarker AXTextMarker::findMarker(AXDirection direction, AXTextUnit textUnit
     // Paragraphs must be handled differently from word + sentence boundaries, as there is no paragraph break iterator.
     // Rather, paragraph boundaries are based on rendered newlines and differences in node editability and block-grouping (through containing blocks).
     if (textUnit == AXTextUnit::Paragraph) {
-        size_t startLineIndex = currentRuns->lineID(runIndex).lineIndex;
-        unsigned offsetInStartLine =  startLineIndex ? offset() - currentRuns->runLengthSumTo(startLineIndex) : offset();
+        unsigned sumToRunIndex = runIndex ? currentRuns->runLengthSumTo(runIndex - 1) : 0;
+        unsigned offsetInStartLine =  runIndex ? offset() - sumToRunIndex : offset();
```

Comment:
`runIndex` seems to be `size_t`. Do we need to make this subtraction conditional? If it's 0, we'll just subtract offset() - 0, which is the same as just returning `offset()`.

Reply:
Ahh good point, this was from when I used to use startLineIndex instead. I'll update this one